Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM rocker/rstudio:4.0.5
 
 LABEL org.opencontainers.image.title="Research Template" \
-      org.opencontainers.image.description="Dev container image for the OpenSAFELY research template" \
-      org.opencontainers.image.source="https://github.com/opensafely-core/research-template-docker" \
-      org.opencontainers.image.licenses="GPL-3.0-or-later" \
-      org.opencontainers.image.authors="OpenSAFELY Team <team@opensafely.org>" \
-      org.opencontainers.image.vendor="OpenSAFELY"
+    org.opencontainers.image.description="Dev container image for the OpenSAFELY research template" \
+    org.opencontainers.image.source="https://github.com/opensafely-core/research-template-docker" \
+    org.opencontainers.image.licenses="GPL-3.0-or-later" \
+    org.opencontainers.image.authors="OpenSAFELY Team <team@opensafely.org>" \
+    org.opencontainers.image.vendor="OpenSAFELY"
```

Comment:
How did you format the Dockerfile? Whilst I understand the urge, if your preferred formatting differs from my preferred formatting, then we're in danger of playing formatting ping-pong 🏓.

Alternatively, if this change ensures the Dockerfile is formatted consistently, then it would be helpful to say so in the commit message.

Reply:
This is VS Code's default markdown formatter being triggered due to my "format on save" setting. On second thoughts I don't actually like this formatting change so I will drop this commit.

Reply:
Thank you 🙂